src/app/modules/consultations/consultation-profile/consultation-public-responses/consultation-public-responses.component.ts

```diff
@@ -77,4 +82,22 @@ export class ConsultationPublicResponsesComponent implements OnInit, AfterViewCh
     });
   }
 
+  getActiveRound(profileData) {
+    if (profileData && profileData.responseRounds) {
+      const responseRounds = profileData.responseRounds;
```

Comment:
@Nishanthh I don't think this is necessary. You already have  responseRounds defined in line 44